contracts/index/IndexToken.sol

```diff
@@ -679,25 +680,27 @@ contract IndexToken is
     }
 
     /**
-     * @notice Get the USD value of tokens in the pool
+     * @notice Get the USD value of tokens in the vault
      * @return The USD value
      */
-    function _getPoolAssetValue() internal view returns (uint256) {
+    function _getVaultAssetValue() internal view returns (uint256) {
         return
             getValueFromAssetAmount(
                 IDetailedERC20(asset).balanceOf(address(this))
             );
     }
 
     /**
-     * @notice Get the USD value of tokens owed to the pool
-     * @dev Tokens from the pool are typically borrowed by the LP Account
-     * @dev Tokens borrowed from the pool are tracked with mAPT
-     * @return The USD value
+     * @notice Get the USD value of tokens owed to the vault
+     * @dev Tokens from the vault are typically borrowed by the LP Account
+     * @return The USD value.  USD prices have 8 decimals.
      */
     function _getDeployedValue() internal view returns (uint256) {
```

Comment:
If we go with the lending terminology mentioned above, this could become `_getLoanedUsdValue`